src/cgroups/v1/devices.rs

```diff
@@ -98,24 +98,123 @@ impl Devices {
+                access: "rw".to_string(),
+            },
+        ]
+        .iter()
+        .for_each(|d| {
+            set_fixture(&tmp, "devices.allow", "").expect("create allowed devices list");
+            set_fixture(&tmp, "devices.deny", "").expect("create denied devices list");
+
+            Devices::apply_device(&d, &tmp).expect("Apply default device");
+            println!("Device: {}", d.to_string());
+            if d.allow {
+                let allowed_content =
+                    std::fs::read_to_string(tmp.join("devices.allow")).expect("read to string");
+                assert_eq!(allowed_content, d.to_string());
+            } else {
+                let denied_content =
+                    std::fs::read_to_string(tmp.join("devices.deny")).expect("read to string");
+                assert_eq!(denied_content, d.to_string());
+            }
+        });
+    }
+
+    quickcheck! {
+        fn property_test_apply_device(device: LinuxDeviceCgroup) -> bool {
+            let tmp = create_temp_dir("property_test_apply_device").expect("create temp directory for test");
+            set_fixture(&tmp, "devices.allow", "").expect("create allowed devices list");
+            set_fixture(&tmp, "devices.deny", "").expect("create denied devices list");
+            Devices::apply_device(&device, &tmp).expect("Apply default device");
+            if device.allow {
+                let allowed_content =
+                    std::fs::read_to_string(tmp.join("devices.allow")).expect("read to string");
+                allowed_content == device.to_string()
+            } else {
+                let denied_content =
+                    std::fs::read_to_string(tmp.join("devices.deny")).expect("read to string");
+                denied_content == device.to_string()
+            }
+        }
+
+        fn property_test_apply_multiple_devices(devices: Vec<LinuxDeviceCgroup>) -> bool {
+            let tmp = create_temp_dir("property_test_apply_multiple_devices").expect("create temp directory for test");
+            devices.iter()
+                .map(|device| {
+                    set_fixture(&tmp, "devices.allow", "").expect("create allowed devices list");
+                    set_fixture(&tmp, "devices.deny", "").expect("create denied devices list");
+                    Devices::apply_device(&device, &tmp).expect("Apply default device");
+                    if device.allow {
+                        let allowed_content =
+                            std::fs::read_to_string(tmp.join("devices.allow")).expect("read to string");
+                        allowed_content == device.to_string()
+                    } else {
+                        let denied_content =
+                            std::fs::read_to_string(tmp.join("devices.deny")).expect("read to string");
+                        denied_content == device.to_string()
+                    }
+                })
+                .fold(true, |a, b| a && b)
```

Comment:
```suggestion
                .all(|is_ok| is_ok)
```

Reply:
Good catch, I was actually looking for this exactly. Much cleaner and more idiomatic.

Reply:
this change has been made in the recent commit